io.itch.billwoo.RiseOfThePenguinsGB.yml

```
@@ -0,0 +1,57 @@
+app-id: io.itch.billwoo.RiseOfThePenguinsGB
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
```

```suggestion
runtime-version: '25.08'
```
Not the latest runtime